### PR TITLE
fix: Metrics documentation

### PR DESCRIPTION
**Pull request by saranyaeu2987**
Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -23,8 +23,13 @@ best way to define metrics in Argo to avoid problems such as [cardinality explos
 
 There are two kinds of metrics emitted by Argo: **controller metrics** and **custom metrics**.
 
-* Controller metrics are metrics that inform on the state of the controller; i.e., they answer the question "What is the state of the controller right now?".
-* Custom metrics are metrics that inform on the state of a Workflow, or a series of Workflows. These custom metrics are defined by the user in the Workflow spec.
+#### Controller metrics 
+Metrics that inform on the state of the controller; i.e., they answer the question "What is the state of the controller right now?".<br />
```
Comment (simster7) [inline docs/metrics.md:27]:
```suggestion
Metrics that inform on the state of the controller; i.e., they answer the question "What is the state of the controller right now?"
```

Remove this extraneous tag?

**Review by simster7 — CHANGES_REQUESTED** [review]

**Comment by simster7** [comment]
Thanks for this PR! You need to sign your commits. Please see: https://github.com/argoproj/argo/pull/4829/checks?check_run_id=1650422948

**Comment by saranyaeu2987** [comment]
@simster7 
Committed and signed. Some others' changes were added on rebase. Can you please review. I can rebase again if needed.

**Review by simster7 — APPROVED** [review]

**Comment by simster7** [comment]
Thanks!

**Pull request merged**